src/theory/strings/theory_strings.h

```diff
@@ -602,6 +608,21 @@ class TheoryStrings : public Theory {
    * effort, the call to this method does nothing.
    */
   void registerTerm(Node n, int effort);
+  //-------------------------------------send inferences
+  /** send internal inferences
+   * 
+   * This is called when we have inferred exp => conc, where exp is a set
+   * of equalities and disequalities that hold in the current equality engine.
+   * This method adds equalities and disequalities ~( s = t ) via
+   * sendInference such that both s (resp. t) is either a constant or a term
```

Comment:
"such that both s (resp. t) is" reads a bit strange, maybe just write "such that both s and t are"